crates/bevy_math/src/cubic_splines.rs

```diff
@@ -968,14 +1047,15 @@ impl<P: Point> RationalSegment<P> {
         Self {
             coeff,
             weight_coeff,
+            knot_span,
         }
     }
 }
 
-/// A collection of [`RationalSegment`]s chained into a curve.
+/// A collection of [`RationalSegment`]s chained into a single parametric curve.
 ///
 /// Use any struct that implements the [`RationalGenerator`] trait to create a new curve, such as
-/// [`CubicNURBS`].
+/// [`CubicNURBS`], or convert [`CubicCurve`] using `into/from`.
```

Comment:
Wrong type name in doc
```suggestion
/// [`CubicNurbs`], or convert [`CubicCurve`] using `into/from`.
```